adx-queries/README.md

```diff
@@ -0,0 +1,78 @@
+
+1. Create an Azure Digital Twins instance using the CLI commands below or via the Azure portal. Copy the instance's URL once it is online.
+
+    ```
+    az account set --subscription <your-subscription-ID>
+    az dt create -n <instance-name> -g <resource-group> 
+    ```
+
+    **Note: Ensure the resource group exists.**
+  
+1. Grant yourself the **Azure Digital Twins Data Owner** role on the instance. It may take up to five minutes for this RBAC change to apply.
+  
+    ```
+    az dt role-assignment create -n <instance-name> --assignee "<owneruser@microsoft.com>" --role "Azure Digital Twins Data Owner" 
+    ``` 
+
+1. Download the sample models and graph from the [energy-grid-example](../models/energy-grid-example) folder to your local computer. You can do this by cloning this repository or downloading it as a ZIP, and then navigating to the energy-grid-example folder on your machine.
+
+1. Open [Azure Digital Twins Explorer](https://explorer.digitaltwins.azure.net/). It should prompt you to enter your Azure Digital Twins instance URL. If not, click on the globe icon in the top right corner.
+
+1. Enter your Azure Digital Twins instance URL (endpoint must start with 'https://').
+
+1. On the left pane, click on the **Upload a Model** button (cloud icon with upward arrow).
+
+    ![Screenshot of the Upload a Model icon in Azure Digital Twins Explorer.](../images/adx-queries/upload-model.png)
+
+    1. Go to the folder where you downloaded the repository. Navigate into *<folder>/model/energy-grid-example*.
+
+    1. Select all the JSON files (models) and upload them.
+
+1. In the middle pane, click the **Import Graph** button (also a cloud icon with upward arrow).
+
+    ![Screenshot of the Import Graph icon in Azure Digital Twins Explorer.](../images/adx-queries/import-graph.png)
+
+    1. Within *<folder>/model/energy-grid-example*, select **distributionGrid.xlsx** to import it.
+
+1. You should see a graph preview. Click on the floppy disk icon to finish importing the graph (this creates twins and relationships).
+
+1. Navigate back to the Twin Graph tab. In the query section, type in "SELECT * FROM DIGITALTWINS" and select Run Query. You should see the graph displayed in the Twin Graph window.
+
+1. Click on one of the leaf nodes that has this model: `dtmi:example:grid:plants:domesticConsumer;1`.
+
+1. Update the PowerRequirement value to 500 and save it (using the disc button circled in red below) to patch the twin.
+
+    ![Screenshot of updating and saving a value in Azure Digital Twins Explorer.](../images/adx-queries/save-patch.png)
+
+## Step 2: Run joint Azure Digital Twins/ADX queries
+
+1. [Connect to the Samples database](https://dataexplorer.azure.com/clusters/help/databases/Samples) on the public ADX help cluster. The database features the table **SamplePowerRequirementHistorizedData** that contains historized property values over a one month period for selected digital twins in the graph.
```

Comment:
Please obscure the image where it shows the name of my ADT instance in the ADX explorer window

Reply:
Removing your name and replacing with a generic sample name, ADTinstance (so readers can still see the format of the value from the screenshot)